beets/library.py

```diff
@@ -1237,14 +1237,17 @@ def __init__(self, path='library.blb',
         timeout = beets.config['timeout'].as_number()
         super(Library, self).__init__(path, timeout=timeout)
 
-        self._connection().create_function('bytelower', 1, _sqlite_bytelower)
-
         self.directory = bytestring_path(normpath(directory))
         self.path_formats = path_formats
         self.replacements = replacements
 
         self._memotable = {}  # Used for template substitution performance.
 
+    def _create_connection(self):
+        conn = super(Library, self)._create_connection()
+        conn.create_function('bytelower', 1, _sqlite_bytelower)
+        return conn
+
```

Comment:
this was my fault for not understanding how create_function worked. I thought it went to all connection automatically :(

Can you submit this as a separate PR? It's a general bug fix.